releases/release-4.0.6.md

```diff
@@ -183,7 +183,7 @@ TiDB version: 4.0.6
         - Fix the issue that `changefeed` might be blocked by MySQL driver [#936](https://github.com/pingcap/ticdc/pull/936)
         - Fix the incorrect Resolved Ts interval of TiCDC [#8573](https://github.com/tikv/tikv/pull/8573)
 
-    + BR
+    + Backup & Restore (BR)
```

Comment:
```suggestion
    + BR
```
I think we only need to spell it out when it first appears in the docs, right? Its first appearance is at L81.

Reply:
In some of the previous release notes, BR is spelled out more than once in each doc. In one or two release notes, it is spelled only once. We need to unify it. IMO, readers might not read notes or titles sequentially, so I prefer spelling it out all.

Reply:
OK~